ann/docker-compose.yml

```diff
@@ -7,18 +7,30 @@ services:
     ports:
       - "3000:3000"
     depends_on:
-      - mysql
+      mysql:
+        condition: service_healthy
     environment:
       MYSQL_HOST: mysql
       MYSQL_PORT: 3306
+      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD}
+      SECRET_KEY: ${SECRET_KEY}
     volumes:
-      - ".:/app"
-      - "/app/node_modules"
+      - "./src:/app/src"
+
   mysql:
-    build:
-      context: .
-      dockerfile: Dockerfile.mysql
+    image: mysql:latest
+    ports:
+      - "3306:3306"
     environment:
       MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD}
+    healthcheck:
+      test: ["CMD", "mysqladmin", "ping", "-h", "localhost"]
+      timeout: 5s
+      interval: 3s
+      retries: 10
     volumes:
-      - "./data:/var/lib/mysql"
\ No newline at end of file
+      - ./musicdatabase.sql:/docker-entrypoint-initdb.d/musicdatabase.sql
+      - mysql:/var/lib/mysql
```

Comment:
We use named volumes for situations like this. Essentailly, docker is mapping a volume somewhere in your file system that is fully controlled by docker and out of the way of your project. More info [here](https://docs.docker.com/compose/compose-file/compose-file-v3/#volume-configuration-reference)